packages/ui/primitives/signature-pad/signature-pad.tsx

```diff
@@ -46,7 +49,7 @@ export const SignaturePad = ({
       end: {
         taper: size * 2,
       },
-    } satisfies StrokeOptions;
+    } as StrokeOptions;
```

Comment:
How come `satisfies` was changed to `as`? 

We try to avoid `as` as much as possible

Reply:
noted.. making a new commit